tests/common/plugins/sanity_check/checks.py

```diff
@@ -43,16 +43,45 @@ def check_services(dut):
     return check_result
 
 
-def _find_down_ports(dut, interfaces):
-    down_ports = []
-    intf_facts = dut.interface_facts()['ansible_facts']
-    for intf in interfaces:
+def _find_down_phy_ports(dut, phy_interfaces):
+    down_phy_ports = []
+    intf_facts = dut.show_interface(command='status', include_internal_intfs=True)['ansible_facts']['int_status']
+    for intf in phy_interfaces:
+        try:
+            if intf_facts[intf]['oper_state'] == 'down':
+                down_phy_ports.append(intf)
+        except KeyError:
+            down_phy_ports.append(intf)
+    return down_phy_ports
+
+
+def _find_down_ip_ports(dut, ip_interfaces):
+    down_ip_ports = []
+    ip_intf_facts = dut.show_ip_interface()['ansible_facts']['ip_interfaces']
```

Comment:
how are you getting "ip interfaces" from all namespace. "show ip interface" module is getting only for 1 namespace.

Reply:
dut here is the SonicAsic. the APi `show_ip_interface `in SonicAsic, we append the namespace and get the data.